src/features/Report/ReportManager.ts

```diff
@@ -1,90 +1,27 @@
+import { FormatService } from './../../common/services/FormatService/FormatService';
```

Comment:
Refactored a little - originally the email function was just a copy paste of the 'write to disk' function which had 'hey also email' at the end. Added types.

Main change is using `parseAsync` and chunking the logs up to parse them all in smaller chunks. Doing one big parse makes it no work .. 🤷